conf/messages.en

```diff
@@ -137,6 +137,17 @@ business.nonUkAddress.checkYourAnswersLabel = What is {0}’s address?
 business.nonUkAddress.title = What is the company’s address?
 business.nonUkAddress.heading = What is {0}’s address?
 
+business.emailYesNo.checkYourAnswersLabel = Do you know {0}’s email address?
```

Comment:
The other related messages for business say "company's" instead of "personal representatives's". Should they be made consistent?